packages/@ourworldindata/components/src/RadioButton.tsx

```diff
@@ -1,6 +1,8 @@
 import * as React from "react"
+import cx from "classnames"
 
 export class RadioButton extends React.Component<{
+    className?: string
```

Comment:
This was the easiest way to fix a type error after I added the `className` to `Checkbox`.